travis_build_test.sh

```diff
@@ -11,6 +11,13 @@ start eslint
 ./node_modules/eslint/bin/eslint.js karma.conf.js app test
 end eslint
 
+start examples
+REGEN_SCRIPT='./docs/examples/regenerate-index.py'
+INDEX_JSON='docs/examples/index.json'
+diff --ignore-space-change <( $REGEN_SCRIPT ) $INDEX_JSON \
```

Comment:
Sorry, I'm having a hard time wrapping my head around what this is doing. Could you explain it in a sentence or two? Or perhaps add a comment?

Reply:
It looks like it doesn't actually update index.json but just warns the user if it needs to be updated?

Reply:
Yep, that's what it does: I'll add a comment. The `<( )` syntax is for process substitution: The output of the command is treated like a file argument that can be read from. (There's also `>( )` for writing.)

Reply:
Merged recent changes from and develop, and fixed this. I'll go ahead and merge if it comes back green.